exercises/say/say_test.rb

```diff
@@ -1,128 +1,116 @@
 require_relative 'say'
 
+# Test data version: e3bd4a2
 class SayTest < Minitest::Test
-  def test_0
-    assert_equal 'zero', Say.new(0).in_english
+  def test_zero
+    # skip
+    question = '0'
+    assert_equal('zero', SayProblem.new(question).answer)
   end
 
   def test_one
     skip
-    assert_equal 'one', Say.new(1).in_english
+    question = '1'
+    assert_equal('one', SayProblem.new(question).answer)
   end
 
-  def test_14
+  def test_fourteen
     skip
-    assert_equal 'fourteen', Say.new(14).in_english
+    question = '14'
+    assert_equal('fourteen', SayProblem.new(question).answer)
   end
 
   def test_twenty
     skip
-    # This really shouldn't be twenty-zero
-    assert_equal 'twenty', Say.new(20).in_english
+    question = '20'
+    assert_equal('twenty', SayProblem.new(question).answer)
   end
 
   def test_twenty_two
     skip
-    assert_equal 'twenty-two', Say.new(22).in_english
+    question = '22'
+    assert_equal('twenty-two', SayProblem.new(question).answer)
   end
 
-  def test_100
+  def test_one_hundred
     skip
-    assert_equal 'one hundred', Say.new(100).in_english
+    question = '100'
+    assert_equal('one hundred', SayProblem.new(question).answer)
   end
 
-  def test_120
+  def test_one_hundred_twenty_three
     skip
-    assert_equal 'one hundred twenty', Say.new(120).in_english
+    question = '123'
+    assert_equal('one hundred twenty-three', SayProblem.new(question).answer)
   end
 
-  def test_123
+  def test_one_thousand
     skip
-    assert_equal 'one hundred twenty-three', Say.new(123).in_english
+    question = '1000'
+    assert_equal('one thousand', SayProblem.new(question).answer)
   end
 
-  def test_1_thousand
+  def test_one_thousand_two_hundred_thirty_four
     skip
-    assert_equal 'one thousand', Say.new(1000).in_english
+    question = '1234'
+    assert_equal('one thousand two hundred thirty-four', SayProblem.new(question).answer)
   end
 
-  def test_1_thousand_234
+  def test_one_million
     skip
-    expected = 'one thousand two hundred thirty-four'
-    assert_equal expected, Say.new(1234).in_english
+    question = '1000000'
+    assert_equal('one million', SayProblem.new(question).answer)
   end
 
-  def test_1_million
+  def test_one_million_two_thousand_three_hundred_forty_five
     skip
-    assert_equal 'one million', Say.new(10**6).in_english
+    question = '1002345'
+    assert_equal('one million two thousand three hundred forty-five', SayProblem.new(question).answer)
   end
 
-  def test_1_million_and_some_crumbs
+  def test_one_billion
     skip
-    assert_equal 'one million two', Say.new(1_000_002).in_english
+    question = '1000000000'
+    assert_equal('one billion', SayProblem.new(question).answer)
   end
 
-  def test_1_million_2_thousand_345
+  def test_a_big_number
     skip
-    expected = 'one million two thousand three hundred forty-five'
-    assert_equal expected, Say.new(1_002_345).in_english
+    question = '987654321123'
```

Comment:
It looks like the previous tests have the `question` as a number, rather than a string.

They also format them to make it a bit easier to read the larger numbers. For example: `987_654_321_123`